### PR TITLE
Implementation of #156

### PR DESCRIPTION
**Pull request by ScoutAtlas**
#156

### CODE REVIEW

File: backend/Backend/Startup.cs [hunk 1]
```diff
@@ -39,7 +39,11 @@ public void ConfigureServices(IServiceCollection services)
             {
                 options.AddPolicy(name: MyAllowSpecificOrigins,
                                   builder => {
-                                      builder.WithOrigins("http://localhost:3000").AllowAnyHeader();
+                                      builder.WithOrigins(
+                                          "http://localhost:3000",
+                                          "http://127.0.0.1:3000",
+                                          "https://localhost:3000",
+                                          "https://127.0.0.1:3000").AllowAnyHeader();
```
Comment (SaiVarunVaranasi) [inline backend/Backend/Startup.cs:47]: Which URL is currently in use to access the website? or We can use any of these URLs to access the website?

**Review by SaiVarunVaranasi — APPROVED** [review]
Looks Good to me. Please share the URLs in Slack as well.

**Review by manifau — COMMENTED** [review]
The titles in the comparison page are adjusted to the current scenario.
image

File: frontend/src/components/cards/CardComponent.js [hunk 2]
```diff
@@ -54,24 +54,24 @@ function CardComponent(props) {
...
     return (
-        <Column
+        <Col
             flexGrow={1}
             className={[classes.container, containerStyles].join(' ')}
             breakpoints={{ 426: classes.containerMobile }}
         >
-            <Row horizontal='space-between'>
-                <Column>
+            <Row justify='space-between'>
```
Comment (Waldleufer) [inline frontend/src/components/cards/CardComponent.js:73]:
```suggestion
            <Row justify='between'>
```

**Review by Waldleufer — APPROVED** [review]
lgtm

**Pull request merged**